network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -1258,17 +1306,17 @@ def exec(self, only_captured=True):
 
 
 class ForbidsQuery(TwoNetworkConfigsQuery):
+    """
+    Checking whether the connections explicitly allowed by config1 are denied by config2
+    """
 
     def exec(self):
         if not self.config1:
             return QueryAnswer(False, 'There are no NetworkPolicies in the given forbids config. '
                                       'No traffic is specified as forbidden.', query_not_executed=True)
-        if self.config1.type == NetworkConfig.ConfigType.Ingress \
-                or self.config2.type == NetworkConfig.ConfigType.Ingress:
-            ingress_name = self.config1.name if self.config1.type == NetworkConfig.ConfigType.Ingress \
-                else self.config2.name
-            return QueryAnswer(bool_result=False,
-                               output_result=f'Ignoring ForbidsQuery for {ingress_name} with Ingress only',
+        if self.config1.layers.does_contain_single_layer(NetworkLayerName.Ingress) or \
+                self.config2.layers.does_contain_single_layer(NetworkLayerName.Ingress):
+            return QueryAnswer(bool_result=False, output_result='Ignoring ForbidsQuery for config with Ingress only',
```

Comment:
Same here. Ingress should not be used only for the definition of denied traffic

Reply:
done